shared/cfg/base.cfg

```diff
@@ -699,7 +699,7 @@ Linux:
     stress_install_from_repo = "no"
     # param for stress tests
     stress_args = '--cpu 4 --io 4 --vm 2 --vm-bytes 256M'
-    download_url_stress = 'http://people.seas.harvard.edu/~apw/stress/stress-1.0.4.tar.gz'
+    download_url_stress = '/home/kar/workspace/avocado-vt/shared/deps/stress/stress-1.0.4.tar.gz'
```

Comment:
https://fossies.org/linux/privat/stress-1.0.4.tar.gz

Reply:
I'm afraid that it will invalid in some day like before, so put it to somewhere in avocado.

Reply:
so for every new release we would have to download and maintain it locally, I would not prefer it, @PaulYuuu what would you suggest.

Reply:
If the current stress version works fine in the test case, I don't think we have to pursue the latest version. And if this version does not suitable for newer guests, it is not very difficult to specify it in the cfg file of the test case.

Reply:
okay, make sense.

Reply:
> I searched the code for a bit, maybe `utils_misc.get_path(data_dir.DEPS_DIR, stress-1.0.4.tar.gz)` is right for us. We can save this package to `avocado-vt/shared/deps/stress/stress-1.0.4.tar.gz`
> And you're right, tp-libvirt has one line about download_url_stress: https://github.com/autotest/tp-libvirt/blob/a7c4001e748f63fe6dfb45aaf5d6d624d25500d9/libvirt/tests/cfg/virsh_cmd/domain/virsh_migrate_stress.cfg#L54
> So it also needs to be modified.
I got `/usr/share/avocado-plugins-vt/shared/deps/stress-1.0.4.tar.gz` for `utils_misc.get_path(data_dir.DEPS_DIR, stress-1.0.4.tar.gz)` , and need I deal with the libvirt issue?

Reply:
yes, I would suggest you send out a patch to tp-libvirt that gets affected by this change